docs/src/app/components/pages/components/List/Page.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import listReadmeText from './README';
+import listCode from '!raw!material-ui/lib/lists/list';
+import listItemCode from '!raw!material-ui/lib/lists/list-item';
+import listExampleSimpleCode from '!raw!./ExampleSimple.jsx';
+import ListExampleSimple from './ExampleSimple.jsx';
+import listExampleChatCode from '!raw!./ExampleChat.jsx';
+import ListExampleChat from './ExampleChat.jsx';
+import listExampleContactsCode from '!raw!./ExampleContacts.jsx';
+import ListExampleContacts from './ExampleContacts.jsx';
+import listExampleFoldersCode from '!raw!./ExampleFolders.jsx';
+import ListExampleFolders from './ExampleFolders.jsx';
+import listExampleNestedCode from '!raw!./ExampleNested.jsx';
+import ListExampleNested from './ExampleNested.jsx';
+//ExampleNested causes warning!
```

Comment:
@oliviertassinari Got it! It was the nestedItems prop, ListItems passed in the array need keys since they are part of an array